demo/src/main/java/com/dooapp/fxform/Demo.java

```diff
@@ -79,7 +78,13 @@ protected void setup() {
     private Node createNode() {
         VBox vBox = new VBox();
         vBox.getChildren().addAll(createSkinSelector(), createCSSNode(), createSnapshotButton(), fxForm, createConstraintNode());
-        return ScrollPaneBuilder.create().content(vBox).fitToWidth(true).build();
+       return new ScrollPane(){
+            {
+                setContent(vBox);
+                setFitToWidth(true);
+            }
+        };
+        //        return ScrollPaneBuilder.create().content(vBox).fitToWidth(true).build();
```

Comment:
Could you please remove these commented builders ? We won't need it anymore.

Reply:
Sure!